algorithms/unit_tests/TestStdAlgorithmsTeamNoneOf.cpp

```diff
@@ -0,0 +1,178 @@
+
+  KOKKOS_INLINE_FUNCTION
+  bool operator()(ValueType val) const { return (val > m_val); }
+};
+
+template <class SourceViewType, class DestViewType, class UnaryOp>
+struct TestFunctorA {
+  SourceViewType m_sourceView;
+  DestViewType m_destView;
+  int m_apiPick;
+  UnaryOp m_unaryOp;
+
+  TestFunctorA(const SourceViewType sourceView, const DestViewType destView,
+               int apiPick, UnaryOp unaryOp)
+      : m_sourceView(sourceView),
+        m_destView(destView),
+        m_apiPick(apiPick),
+        m_unaryOp(std::move(unaryOp)) {}
+
+  template <class MemberType>
+  KOKKOS_INLINE_FUNCTION void operator()(const MemberType& member) const {
+    const auto myRowIndex = member.league_rank();
+
+    auto myRowViewFrom =
+        Kokkos::subview(m_sourceView, myRowIndex, Kokkos::ALL());
+
+    switch (m_apiPick) {
+      case 0: {
+        const bool result = KE::none_of(member, KE::cbegin(myRowViewFrom),
+                                        KE::cend(myRowViewFrom), m_unaryOp);
+        Kokkos::single(Kokkos::PerTeam(member),
+                       [=]() { m_destView(myRowIndex) = result; });
+        break;
+      }
+
+      case 1: {
+        const bool result = KE::none_of(member, myRowViewFrom, m_unaryOp);
+        Kokkos::single(Kokkos::PerTeam(member),
+                       [=]() { m_destView(myRowIndex) = result; });
+        break;
+        break;
+      }
+    }
+  }
+};
+
+template <class LayoutTag, class ValueType>
+void test_A(std::size_t numTeams, std::size_t numCols, int apiId) {
+  /* description:
+     use a rank-2 view randomly filled with values,
+     and run a team-level none_of
+   */
+
+  // -----------------------------------------------
+  // prepare data
+  // -----------------------------------------------
+  // create a view in the memory space associated with default exespace
+  // with as many rows as the number of teams and fill it with random
+  // values from an arbitrary range.
+  constexpr auto lowerBound = ValueType{5};
+  constexpr auto upperBound = ValueType{523};
+  Kokkos::pair bounds{lowerBound, upperBound};
+  auto [sourceView, sourceViewBeforeOp_h] = create_random_view_and_host_clone(
+      LayoutTag{}, numTeams, numCols, bounds, "sourceView");
+
+  // -----------------------------------------------
+  // launch kokkos kernel
+  // -----------------------------------------------
+  using space_t = Kokkos::DefaultExecutionSpace;
+  Kokkos::TeamPolicy<space_t> policy(numTeams, Kokkos::AUTO());
+
+  // create the destination view
+  Kokkos::View<bool*> destView("destView", numTeams);
+
+  GreaterThanValueFunctor unaryPred{upperBound};
+
+  // use CTAD for functor
+  TestFunctorA fnc(sourceView, destView, apiId, unaryPred);
+  Kokkos::parallel_for(policy, fnc);
+
+  // -----------------------------------------------
+  // run cpp-std kernel and check
+  // -----------------------------------------------
+  auto destView_h = create_host_space_copy(destView);
```

Comment:
not needed/used

Reply:
It is needed - none_of returns bool. I renamed it to noneOfResultsView.

Reply:
Done.